storybook/app/builtin/stories/clear-refinements.stories.js

```diff
@@ -27,25 +27,43 @@ export default () => {
       )
     )
     .add(
-      'with nothing to clear',
+      'with query only (via includedAttributes)',
+      wrapWithHits(container => {
+        window.search.addWidget(
+          instantsearch.widgets.clearRefinements({
+            container,
+            includedAttributes: ['query'],
+            templates: {
+              resetLabel: 'Clear query',
+            },
+          })
+        );
+      })
+    )
+    .add(
+      'with query only (via excludedAttributes)',
       wrapWithHits(container => {
         window.search.addWidget(
           instantsearch.widgets.clearRefinements({
             container,
+            excludedAttributes: [],
+            templates: {
+              resetLabel: 'Clear refinements and query',
+            },
           })
         );
       })
     )
     .add(
-      'with clear refinements and query',
+      'with refinements and query',
```

Comment:
this is duplicate right?

Reply:
Duplicate with which one? I updated the stories anyway.